test/run-php-linter.sh

```diff
@@ -13,26 +13,69 @@ vendor/bin/phpcs --standard=docs/LorisCS.xml php/libraries php/exceptions php/in
-vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/media
-vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/candidate_parameters/ajax
+vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/data_team_helper
+vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/datadict 
+vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/dataquery
 vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/dicom_archive
-vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/create_timepoint
+vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/document_repository
+vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/examiner
+vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/final_radiological_review
+vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/genomic_browser
+vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/help_editor 
+vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/imaging_browser
+vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/imaging_uploader
+vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/instrument_builder
+vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/instrument_list 
+vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/instrument_manager
 vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/issue_tracker
+vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/media 
+vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/mri_violations
+vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/new_profile 
+vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/next_stage
+vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/reliability
+vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/server_processes_manager
+vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/statistics
+vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/survey_accounts
+vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/timepoint_flag
+vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/timepoint_list
+vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/training 
+vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/user_accounts
```

Comment:
Can all these lines be replaced by just one:

```
vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules
```

line? And I think the return code needs to be checked since this is a separate script and not in .travis.yml, to ensure that the line in the .travis.yml returns a non-zero error code when there's an error.